README.md

```diff
@@ -28,10 +28,12 @@ Additional description about the project and its features.
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+none
 
 ### Setup
 
 ### Install
+clone the repo
```

Comment:
- [ ] Please make sure to include clear, step-by-step instruction on how to setup and run the project locally. Just saying "clone the repo" is not enough; what do I do after I clone the repo? Please be clear and concise.